src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -0,0 +1,815 @@
+/*
```

Comment:
Interface? Really? What's wrong with Abstract?

Reply:
I wanted to force following methods to implement each of these steps in their own way, wait, assert, verify. I figured this is a good way to enforce alignment